pkg/kubelet/kubelet.go

```diff
@@ -319,6 +327,26 @@ type Dependencies struct {
 	useLegacyCadvisorStats bool
 }
 
+// newCrashLoopBackOff configures the backoff maximum to be used
+// by kubelet for container restarts depending on the alpha gates
+// and kubelet configuration set
+func newCrashLoopBackOff(kubeCfg *kubeletconfiginternal.KubeletConfiguration) (time.Duration, time.Duration) {
+	boMax := MaxCrashLoopBackOff
+	boInitial := containerBackOffPeriod
+	if utilfeature.DefaultFeatureGate.Enabled(features.ReduceDefaultCrashLoopBackOffDecay) {
+		boMax = reducedMaxCrashLoopBackOff
+		boInitial = reducedInitialCrashLoopBackOff
+	}
+
+	if utilfeature.DefaultFeatureGate.Enabled(features.KubeletCrashLoopBackOffMax) {
+		boMax = kubeCfg.CrashLoopBackOff.MaxContainerRestartPeriod.Duration
+		if boMax < containerBackOffPeriod {
```

Comment:
```suggestion
		if boMax < boInitial {
```
right?

Reply:
Thanks for the catch -- this made me realize the Conflict Resolution section in the KEP had an error (which was the basis for the table tests) so see also https://github.com/kubernetes/enhancements/pull/5201